chirpstack/Cargo.toml

```diff
@@ -234,10 +234,22 @@
       "/etc/chirpstack/region_us915_6.toml",
       "/etc/chirpstack/region_us915_7.toml",
     ]
+    provides = "chirpstack"
     suggests = "postgresql, mosquitto, redis"
     maintainer-scripts = "debian/"
     systemd-units = { enable = true }
 
+  [package.metadata.deb.variants.postgres]
+    name = "chirpstack"
+    suggests = "postgresql, mosquitto, redis"
+    conflicts = "chirpstack-sqlite"
+
+  [package.metadata.deb.variants.sqlite]
+    default-features = false
+    features = ["sqlite"]
+    suggests = "sqlite3, mosquitto, redis"
```

Comment:
Actually `sqlite3` is statically linked, so no need to include it :-)

Reply:
Yes, I know. This was to be consistent with the PostgreSQL package, which is also not needed (libpq is).

This is the `Suggests:` section. If you prefer, I can remove `sqlite3` from the suggested packages.